modules/apps/portal/portal-tika-test/src/testIntegration/resources/com/liferay/portal/tika/test/dependencies/test-x-matlab.js

```diff
@@ -0,0 +1,2 @@
+function a() {
```

Comment:
What is this?

Reply:
🤔

Reply:
Please see the integration test.

I use this fake .js file for testing porpouses